gradle/lib/dependabot/gradle/package/package_details_fetcher.rb

```diff
@@ -83,7 +85,7 @@ def fetch_available_versions
 
             package_releases << {
               version: Gradle::Version.new(version),
-              released_at: release_date_info.none? ? nil : (release_date_info[version]&.fetch(:release_date) || nil),
+              released_at: info[:released_at] || release_date_info[version]&.fetch(:release_date),
```

Comment:
This change should be equivalent to the code before, except that now if `released_at` is set directly in `info` (only applies for distributions) it will take priority. This was to avoid duplicating the `release_date_info` structure from Maven packages